src/helpers/ast/is-literal.js

```diff
@@ -0,0 +1,4 @@
+// Literals, and `undefined`, are literals
+export default function isLiteral(t, node) {
+  return t.isLiteral(node) || (t.isIdentifier(node) && node.name === "undefined");
+}
```

Comment:
Yeah but you're conflating two different AST node types. I read `isLiteral` and that means something very different than `isLiteralOrUndefined` and I wouldn't ever expect it to be because `Literal` means something very specific.